Tests/Repository/Vcs/GitDriverTest.php

```diff
@@ -102,6 +102,66 @@ public function testPublicRepositoryWithEmptyComposer($type, $filename)
      * @param string $type
      * @param string $filename
      */
+    public function testPublicRepositoryWithSkipUpdate($type, $filename)
+    {
+        $repoUrl = 'https://github.com/fxpio/composer-asset-plugin.git';
+        $identifier = '92bebbfdcde75ef2368317830e54b605bc938123';
+        $io = $this->getMockBuilder('Composer\IO\IOInterface')->getMock();
+
+        $this->config->merge([
+            'config' => [
+                'fxp-asset' => [
+                    'git-driver' => [
+                        'skip-update' => '1 week',
+                    ],
+                ],
+                'cache-vcs-dir' => sys_get_temp_dir().'/composer-test-cache-vcs',
+            ],
+        ]);
+
+        $repoConfig = array(
+            'url' => $repoUrl,
+            'asset-type' => $type,
+            'filename' => $filename,
```

Comment:
Can I do this in `setUp()` in the test for all tests? 
When using the RepositoryManager in `GitDriver`, most of the tests fail with...

```
6) Fxp\Composer\AssetPlugin\Tests\Repository\Vcs\GitDriverTest::testPublicRepositoryWithFilesystemCache with data set #1 ('bower', 'bower.json')
Undefined index: asset-repository-manager

/app/Repository/Vcs/GitDriver.php:46
/app/Tests/Repository/Vcs/GitDriverTest.php:249

7) Fxp\Composer\AssetPlugin\Tests\Repository\Vcs\GitHubDriverTest::testPrivateRepositoryNoInteraction with data set #0 ('npm', 'package.json')
Error: Call to a member function get() on null
```

But the actual code work, this is just an issue during testing.